moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -1026,4 +1026,22 @@ bool TimeOptimalTrajectoryGeneration::computeTimeStamps(robot_trajectory::RobotT
 
   return true;
 }
+
+bool TimeOptimalTrajectoryGeneration::computeTimeStamps(
```

Comment:
This function doesn't really look specific to TOTG at all. Why not just append the trajectories first and call the existing one?

Reply:
That's a good point. Assuming people can find the API for that. I'll just close this.